apps/gp2-frontend/src/shared/state.ts

```diff
@@ -10,7 +10,7 @@ const keywordsSelector = selector({
     return getKeywords(authorization);
   },
 });
-const keywordsState = atom<gp2.ListKeywordsResponse>({
+const keywordsState = atom<gp2.ListTagsResponse>({
```

Comment:
same with state and selector above

Reply:
About these changes in files that are not tests, I would rather to do it a future PR, because this change would cause a lot of changes in other parts of FE and this PR is already too big.

Reply:
I've added a checkbox "More renames in FE not covered in this PR" in the description along with the things I intend to do in the next PR 😉 

 and I've added this commit https://github.com/yldio/asap-hub/pull/3745/commits/b0773ef5174bfb4282847cf0a26b328dc74dc505 to this PR with the changes that are only affected in is own file (that I don't need to make changes in a series of other files).